src/nvim/api/private/defs.h

```diff
@@ -129,5 +129,9 @@ struct key_value_pair {
   Object value;
 };
 
+typedef struct {
+  bool bold, underline, undercurl, italic, reverse;
+  int foreground, background, special;
+} HlAttrs;
```

Comment:
why is this in API? Perhaps move it to globals.h if the old location caused a cycle. Or `ui_defs.h`.